index.js

```diff
@@ -1,10 +1,8 @@
 'use babel';
 
 import JSCS from 'jscs';
-import { Range } from 'atom';
-import { findFile } from 'atom-linter';
-import { readFileSync } from 'fs';
-import stripJSONComments from 'strip-json-comments';
+import path from 'path';
+import configFile from './node_modules/jscs/lib/cli-config';
```

Comment:
@acusti I learned that it should work like this:
`import configFile from 'jscs/lib/cli-config'`
_as learned here:_
https://github.com/jscs-dev/node-jscs/issues/1673